client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -166,7 +166,7 @@ def test_download_manifest(self, client, in_tempdir):
             min_replicas=1, max_replicas=10, min_scale=0.1, max_scale=2
         )
         autoscaling.add_metric(CpuUtilizationTarget(0.75))
-        env_vars = {"env1": "var1", "env2": "var2"}
+        env_vars = {"ENV1": "var1", "ENV2": "var2"}
```

Comment:
This fixes

```
FAILED test_endpoint/test_endpoint.py::TestEndpoint::test_download_manifest - requests.exceptions.HTTPError: 422 Client Error: update.env.0.name in body should match '^[A-Z][A-Z0-9_]*$' 
```

Basically, it seems we added validation to ensure that environment variable names are all caps.

Reply:
I have a very vague memory of this

Reply:
I did. Plus other validations. After it accepted a white space and I spent a lot of time figuring out why something was broken.